controller/commentsController.js

```diff
@@ -51,3 +52,37 @@ exports.flagComment = async (req, res) => {
     errHandler(error, res);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.commentBody;
+  if (!commentBody || commentBody.trim() == "") {
+    return res.status(400).json({ message: "Invalid comment body" });
+  }
+  Comments.findById(comment_id)
+    .exec()
+    .then((comment) => {
+      if (!comment) {
+        return next(new CustomError(404, "Comment not found!!"));
```

Comment:
Please use responseHandler. The CustomError is for when there is an errror thrown.